apps/web/.env

```diff
@@ -1,6 +0,0 @@
-# DO NOT ADD SECRETS TO THIS FILE. This is a good place for defaults.
-# If you want to add secrets use `.env.local` instead.
-
```

Comment:
they probably don't want you to delete this file. Instead use `.env.local` as they suggested. Next will pick up both files and combine them. That way you are changing less files also (can revert .gitignore as well)